autoload/tsuquyomi.vim

```diff
@@ -286,6 +299,22 @@ function! s:sortTextComparator(entry1, entry2)
   endif
 endfunction
 
+function! tsuquyomi#signatureHelp()
+  if len(s:checkOpenAndMessage([expand('%:p')])[1])
+    return
+  endif
+
+  pclose
+
```

Comment:
You should call `s:flush()` before calling tsserver method.

If user changes the current buffer after the last calling tsserver's API, there may be some differences between the script source recognized by tsserver with the current buffer. `s:flush` function synchronize the script source on tsserver with the source code of the current buffer if it's dirty.